en/micro-integrator/docs/setup/deployment/k8s-samples/jms-sender-receiver.md

```diff
@@ -260,36 +236,46 @@ Finally, the created Maven Multi Module project should look as follows:
     class="org.apache.axis2.transport.jms.JMSSender"
     ```  
 
-## Step 3: Package and build the solution 
+## Step 3: Build and Push the Docker image
     
-You need to build a Docker image of the integration solution and push it to your Docker registry.
+There are two ways to build a Docker image of the integration solution and push it to your Docker registry:
+
+-   Using <b>Maven</b>:
+
+    1.  Navigate to the Integration project and run the following command to build the project. It will create a docker image with the provided target repository and tag once the build is successfull.
+
+        ```bash
+        mvn clean install -Dmaven.test.skip=true
```

Comment:
With the new project pom structure in IS 7.1.0 release, we can build and push docker images from a single command. 
```
mvn clean install -Dmaven.test.skip=true -Ddockerfile.username={username} -Ddockerfile.password={password} 
```